### PR TITLE
Rr style welcome page

### LINKED ISSUE #59: 27. As a user I want to see a fun, inviting page when I first access the application
Adding style to the welcome page with a fun background image and some colors will make the experience more interesting for the user, and using Material UI components will add uniformity throughout the entire app. 

AC: 

- The user sees a background image instead of a lot of white space
- The user sees the app name
- The user sees fun colors rather than just black and white
- The user sees elements (like the buttons, inputs) that are the same throughout the app

### PR DESCRIPTION
**Pull request by rachelrohrbach**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description
This PR adds styling to the welcome page including a background image, color, and uses the Material UI components `Box`, `Button`, `Container`, and `TextField`. 

## Related Issue
closes #59 

## Acceptance Criteria
- [ ] The user sees a background image instead of a lot of white space
- [ ] The user sees the app name
- [ ] The user sees fun colors rather than just black and white
- [ ] The user sees elements (like the buttons, text inputs) that are the same throughout the app

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
### Before
welcomeScreenshotBefore

### After
welcomeScreenshot

## Testing Steps / QA Criteria
* One way to test the site is to click on the `details` button below to open the `Netlify preview`
Screen Shot 2021-01-12 at 6 05 51 PM

* Otherwise from your terminal, pull down this branch with `git pull origin RR-style-welcome-page` and check that branch out with `git checkout RR-style-welcome-page`
* Run `npm start` in your terminal to launch the application in the browser and see the welcome page

### CODE REVIEW

**Comment by laurenmbeatty** [comment]
This is sooooo pretty. When I switch down to mobile view, there's a little overflow happening. Maybe you want to either tweak the styles or have it flex into a column at this point (like this but maybe with some spacing adjustments):

image

File: src/components/Welcome.js [hunk 1]
```diff
@@ -1,14 +1,47 @@
...
-      <button onClick={onClick}>Create a New List</button>
-      <TokenForm setToken={setToken} />
-    </div>
+    <Box className="background">
+      <h2>
+        <img
+          src={welcomeHeader}
+          alt="welcome to smart shopper"
+          id="welcome-header"
+        ></img>
```
Comment (laurenmbeatty) [inline src/components/Welcome.js:16]: Can this be a self-closing tag?
```suggestion
       / >
```

**Comment by veeteeran** [comment]
@rachelrohrbach Way to go on the welcome page! It looks so good!!!

File: src/components/TokenForm.js [hunk 2]
```diff
@@ -23,14 +25,26 @@ const TokenForm = ({ setToken }) => {
...
   return (
     <React.Fragment>
-      <div>
-        <h3>Share Token</h3>
+      <Box
+        display="flex"
+        flexDirection="column"
+        alignItems="center"
+        justifyContent="space-around"
+        className="join-list-container"
```
Comment (veeteeran) [inline src/components/TokenForm.js:33]: I don't this we are using this class. Can it be deleted?

File: src/components/TokenForm.js [hunk 3]
```diff
@@ -23,14 +25,26 @@ const TokenForm = ({ setToken }) => {
...
+      >
+        <h2 className="join-list-header">Join Existing List</h2>
         <form onSubmit={handleFormSubmit}>
-          <label htmlFor="token">
-            <input
+          <Box
+            display="flex"
+            flexDirection="column"
+            justifyContent="space-between"
+            className="join-form"
```
Comment (veeteeran) [inline src/components/TokenForm.js:41]: We can probably delete this className also.

File: src/styles/Welcome.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+#welcome-header {
+  background-color: #ffffff;
+  margin: 1rem;
+  padding: 1.5rem;
+  border: lightgrey solid;
+  border-radius: 6px;
+}
+
+.create-button {
```
Comment (veeteeran) [inline src/styles/Welcome.css:28]: I don't think we need this CSS rule. I've commented it out and there's no noticeable change

**Review by veeteeran — APPROVED** [review]

**Comment by veeteeran** [comment]
Way to go on the welcome page! It looks so good!!! 😄

File: src/styles/Welcome.css (hunk 4 above)
Comment (rachelrohrbach) [reply src/styles/Welcome.css:28]: @veeteeran did it not stretch the Create List button when you commented it out?

Comment (rachelrohrbach) [reply src/styles/Welcome.css:28]: @veeteeran did it not stretch the Create List button when you commented it out?

**Review by laurenmbeatty — APPROVED** [review]
LGTM 👍 Nice!

**Review by LINDGRENBA — APPROVED** [review]
I think it looks great. I don't think I have anything to add on top of @laurenmbeatty and @veeteeran 's suggestions. Nice job @rachelrohrbach !

**Pull request merged**